package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Profile management for Bedrock Web Apps",
   "main": "index.js",
   "scripts": {
-    "generate-readme": "jsdoc2md -t template.hbs *.js > README.md"
+    "generate-readme": "jsdoc2md -t template.hbs *.js > README.md",
+    "test": "cd test && npm test"
```

Comment:
Mainly because this doesn't actually work without also doing an `npm install` we don't have this in the root of packages.  I suggest removing it.

Reply:
I actually prefer having this, but I have removed it here: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/26/commits/46217494cc4204f88b31c657b0d3a56572fcdec3